lib/firmata.js

```diff
@@ -1229,10 +1230,19 @@ Board.prototype._sendOneWireRequest = function(pin, subcommand, device, numBytes
 
 Board.prototype.setSamplingInterval = function(interval) {
   var safeint = interval < 10 ? 10 : (interval > 65535 ? 65535 : interval); // constrained
+  this.settings.samplingInterval = safeint;
   this.transport.write(new Buffer([START_SYSEX, SAMPLING_INTERVAL, (safeint & 0xFF), ((safeint >> 8) & 0xFF), END_SYSEX]));
 };
 
 /**
+ * Get sampling interval in millis. Default is 19 ms
+ */
+
+Board.prototype.getSamplingInterval = function(interval) {
+  return this.settings.samplingInterval;
+};
+
```

Comment:
+1 to ditching the get and set style.

1. Would it make sense to deprecate `setSamplingInterval` and add a fluent style `samplingInterval`?
2. Done